dipy/denoise/noise_estimate.py

```diff
@@ -58,7 +53,7 @@ def piesno(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5, return_mask=False)
     sigma : float
         The estimated standard deviation of the gaussian noise.
 
-    mask (optional): ndarray
```

Comment:
Suggested by elef, it's useful to sanity check if your noise estimation is valid by looking at the mask of identified noisy pixel, but I think people mistook it for a brain mask instead and where passing this mask to any other function requiring a mask. If you invert it, you get a bunch of random voxels in the background, so it's not made for segmenting the brain and was a source of confusion.

To prevent that, it's an optional return value. No idea why the (optional) went away though.